src/sound/xaudio2_s.cpp

```diff
@@ -57,6 +57,9 @@ class StreamingVoiceContext : public IXAudio2VoiceCallback
 		this->buffer.resize(buffer_length);
 	}
 
+	/* To silence a clang-cl warning. */
```

Comment:
Is this a relevant comment? It's less "to silence a warning" and more "to make the class definition correct" (yes, aside from the whole notable thing)